subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/InitialPassStatementTransformer.java

```diff
@@ -97,6 +99,21 @@ public Statement transform(SourceUnit sourceUnit, Statement statement) {
                 }
                 seenPluginRepositoriesBlock = true;
                 return statement; // don't transform the pluginRepositories block
+            } else if (scriptBlock.getName().equals(PLUGIN_MANAGEMENT)) {
```

Comment:
The idea was that we would at some point allow the `plugins {}` block in a settings script (so  `Plugin<Settings>` can be applied with the same convenience as `Plugin<Project>`). For that to work the plugin management would have to be configured before the imperative part of the settings script is compiled.